bindata/tnfdeployment/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: tnf-operator
+    control-plane: tnf-controller-manager
+  name: tnf-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: tnf-controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: tnf-controller
+        openshift.io/required-scc: "privileged"
+      labels:
+        control-plane: tnf-controller-manager
+    spec:
+      containers:
+        - name: tnf-controller
+          image: quay.io/slintes/tnf-operator:latest
+          imagePullPolicy: Always
+          command: ["tnf-operator", "operator"]
+          resources:
+            requests:
+              cpu: 50m
+              memory: 64Mi
+            limits:
+              cpu: 500m
+              memory: 128Mi
+          securityContext:
+            privileged: true
+            allowPrivilegeEscalation: true
```

Comment:
do you guys really need root privs for this?

Reply:
the CEO namespace can't as far as I understand your design doc, but can the `openshift-etcd` host those? 
Creating a new namespace with those settings is going to set off a whole barrage of security scanners that I don't want in my bug backlog :/